doc-content/enterprise-only/release-notes/rn-740-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,125 @@
+* `OutOfMemoryError` occurs when starting a high number of process instances with timers [https://issues.jboss.org/browse/RHPAM-2106[RHPAM-2106]]
+
+== Process designer
+* {CENTRAL} process designer prints errors to the server log for the simplest process [https://issues.jboss.org/browse/RHPAM-1782[RHPAM-1782]]
+* Sequence flows from events and gateways are losing endpoint positions [https://issues.jboss.org/browse/RHPAM-1977[RHPAM-1977]]
+* An exception occurs when setting the ruleflow group to a business rule task [https://issues.jboss.org/browse/RHPAM-445[RHPAM-445]]
+* Labels for events and gateways override the element [https://issues.jboss.org/browse/RHPAM-1887[RHPAM-1887]]
+
+== {KIE_SERVER}
+* A 404 error occurs after an OPTIONS request on Smart Router [https://issues.jboss.org/browse/RHPAM-2013[RHPAM-2013]]
+//* AsyncMode True causes an rror with multi-nstance sub processes [https://issues.jboss.org/browse/RHPAM-1968[RHPAM-1968]]
+* PostgreSQL XA datasources do not have a URL method [https://issues.jboss.org/browse/RHPAM-1997[RHPAM-1997]]
+* Process diagram images are hard coded to large sizes [https://issues.jboss.org/browse/RHPAM-1919[RHPAM-1919]]
+
+//== Upgrade tool
+//* License information is not updated when applying a patch [https://issues.jboss.org/browse/RHPAM-1388[RHPAM-1388]]
+
+endif::[]
+
+ifdef::DM[]
+== {CENTRAL}
+* Every rule-based test scenario fails with "Impossible to find a KieSession with name default" [https://issues.jboss.org/browse/RHDM-981[RHDM-981]]
+* A corrupted test scenario occurs when users accept error suggestions for collection [https://issues.jboss.org/browse/RHDM-995[RHDM-995]]
+* Validate all XSD validation only locally  [https://issues.jboss.org/browse/RHDM-973[RHDM-973]]
+* All line separators in RHS in a spreadsheet are escaped [https://issues.jboss.org/browse/RHDM-903[RHDM-903]]
+//* Custom data objects are not shown in guided rules  [https://issues.jboss.org/browse/RHDM-884[RHDM-884]]
+* Drools Test Scenario Audit Log content does not fit in the table view  [https://issues.jboss.org/browse/RHDM-911[RHDM-911]]
+* Cannot resize a column in a guided rule template   [https://issues.jboss.org/browse/RHDM-926[RHDM-926]]
+* Localize the  `library.properties` file  [https://issues.jboss.org/browse/RHDM-688[RHDM-688]]
+* Read-only user can delete fields in Data Object  [https://issues.jboss.org/browse/RHDM-756[RHDM-756]]
+
+== Business Optimizer
+* Employee Rostering shows an exception message when deleting an entity that is already referenced from some other entity  [https://issues.jboss.org/browse/RHDM-919[RHDM-919]]
+* Unable to deploy the out-of-the-box Employee Roster starter application WAR file  [https://issues.jboss.org/browse/RHDM-923[RHDM-923]]
+
+== Decision engine
+* A build error occors with executable-model, nested property with double-bytes property name  [https://issues.jboss.org/browse/RHDM-899[RHDM-899]]
```

Comment:
"with the executable model [end]" No hyphen, and nix everything from the comma on.